practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -54,6 +62,27 @@ const Analytics = () => {
     handleToggleForm();
   }, []);
 
+  // Open or close delete popup
+  const handleToggleDeletePopup = useCallback(() => {
+    setIsShowConfirmDelete(!isShowConfirmDelete);
+  }, [isShowConfirmDelete]);
+
+  // Handle delete customer
+  const handleDeleteCustomer = useCallback(async () => {
+    let toastMessage = MESSAGES.DELETE.SUCCESS;
+    const response = await deleteCustomerService(selectedCustomer.id);
+    if (selectedCustomer) {
```

Comment:
why need to check this has selectedCustomer?

Reply:
Because I want to make sure that if having selectedCustomer, it will take the action below
But I think it is unnecessary in this case, so I changed 
![image](https://github.com/datdragon777/React-Training/assets/74806826/75d7b676-2341-486f-8469-466b0fb51a18)